src/Renderer/AirspaceLabelRenderer.cpp

```
@@ -83,9 +241,67 @@ AirspaceLabelRenderer::DrawInternal(Canvas &canvas,
+      if (cluster == nullptr) {
+        if (clusters.full()) {
+#ifndef NDEBUG
+          LogFmt("AirspaceLabelRenderer: skipped NOTAM label at {},{} "
+                 "(cluster buffer full)",
+                 pos->x, pos->y);
+#endif
+          continue;
+        }
+
+        cluster = &clusters.append();
+        cluster->anchor = *pos;
```

Cppcheck (2.20.0)

[error] Using pointer that is a temporary.

(danglingTemporaryLifetime)